app/views/occupation_standards/_header.html.erb

```diff
@@ -6,12 +6,14 @@
     <!-- <h1 class="page-title" mb-3 text-3xl font-bold leading-none tracking-tight text-gray-100 md:text-4xl xl:text-5xl">Page Title</h1> -->
 
     <! -- If Occupation Standards Page -->
+  <% if current_page?(home_page_path) %>
```

Comment:
@jenlu33 this makes us lose the search form on the occupation standards page. Can we do:

```
<% unless current_page?(terms_page_path) %>
```

instead?